src/web/nextui/src/app/components/InfoModal.tsx

```diff
@@ -0,0 +1,65 @@
+import BugReportIcon from '@mui/icons-material/BugReport';
+import CalendarTodayIcon from '@mui/icons-material/CalendarToday';
+import ForumIcon from '@mui/icons-material/Forum';
+import GitHubIcon from '@mui/icons-material/GitHub';
+import MenuBookIcon from '@mui/icons-material/MenuBook';
+import { Box, Modal, Typography } from '@mui/material';
+import Link from 'next/link';
+import './InfoModal.css';
+
+export function InfoModal({ open, onClose }: { open: boolean; onClose: () => void }) {
+  return (
+    <Modal open={open} onClose={onClose} className="info-modal">
+      <Box className="info-modal-content">
+        <Typography component="h2" variant="h5" className="info-modal-title">
```

Comment:
`variant` is enough, don't need `component`

but this should be replaced with `DialogTitle`